modules/fedimint-ln-client/src/receive.rs

```diff
@@ -266,23 +278,6 @@ impl LightningReceiveConfirmedInvoice {
 
         global_context.claim_input(dbtx, client_input).await
     }
-
-    async fn await_payment_timeout(timeout: Duration) {
-        // Add 10% of the invoice expiry_time as a buffer before we stop awaiting the
-        // payment
-        let timeout_buffer = timeout.as_secs_f64() * 0.1;
```

Comment:
I think having a certain percentage of grace period (previously 10%) is good since clocks aren't perfectly synchronized. Also the contract might get funded right before timeout and we'd just ignore it.